### PR TITLE
Use myext_pi1 instead of my_ext_pi1

### PR DESCRIPTION
**Pull request by sypets**
On the TCEFORM page there are several examples for setting properties in Flexforms. Usually "my_ext_pi1" is given, but this should rather by "myext_pi1".

Resolves: #403

### CODE REVIEW

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 1]
```diff
@@ -178,6 +178,16 @@ The TypoScript syntax to
 :ref:`modify <t3coreapi:typoscript-syntax-syntax-value-modification>` values
 can also be used for the page TSconfig.
 
+Overriding Flexform settings
```
Comment (brotkrueml) [inline Documentation/UsingSetting/PageTSconfig.rst:181]:
```suggestion
Overriding FlexForm settings
```

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 2]
```diff
@@ -178,6 +178,16 @@ The TypoScript syntax to
...
+============================
+
+It is possible to override some FlexForm settings via TSconfig. You can find
```
Comment (brotkrueml) [inline Documentation/UsingSetting/PageTSconfig.rst:184]:
```suggestion
It is possible to override some :ref:`FlexForm <t3coreapi:flexforms>` settings via TSconfig. You can find
```

File: Documentation/UsingSetting/PageTSconfig.rst [hunk 3]
```diff
@@ -178,6 +178,16 @@ The TypoScript syntax to
...
+some examples for this in the reference section, for example on the
+:ref:`TCEFORM <tceform>` page.
+
+The syntax is similar as when overriding TCA settings, but a little different:
```
Comment (brotkrueml) [inline Documentation/UsingSetting/PageTSconfig.rst:188]:
```suggestion
The syntax is similar as when overriding TCA settings, but a little different.
```

**Review by brotkrueml — APPROVED** [review]

**Comment by brotkrueml** [comment]
@sypets Good catch! Thanks for your contribution :+1:

**Comment by sypets** [comment]
@brotkrueml I accidentally left the section [Documentation/UsingSetting/PageTSconfig.rst](https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TSconfig/pull/406/files#diff-575fd5dfa759f5298ec9b1f445c4a7c4d5929e4119284144d520ba1a27c9865d)  in here. I noticed afterwards, there is already a description in TCEFORM at the top of the page. 

I will add another PR to remove it again. Sorry.

**Pull request merged**